src/PlusDataCollection/PointGrey/vtkPlusSpinnakerVideoSource.cxx

```diff
@@ -0,0 +1,807 @@
+/*=Plus=header=begin======================================================
+Program: Plus
+Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+See License.txt for details.
+=========================================================Plus=header=end*/
+
+#include "PlusConfigure.h"
+#include "vtkPlusSpinnakerVideoSource.h"
+
+// Local includes
+#include "vtkPlusChannel.h"
+#include "vtkPlusDataSource.h"
+
+// Spinnaker API includes
+#include "Spinnaker.h"
```

Comment:
total preference, change not needed, but I like to do external includes with `<...>` instead of `"..."`